modern/src/main/StatusCard.js

```diff
@@ -157,7 +157,7 @@ const StatusCard = ({ deviceId, onClose }) => {
     <>
       {device && (
         <Draggable
-          handle={`.${classes.header}`}
+          handle={deviceImage ? `.${classes.media}` : `.${classes.header}`}
```

Comment:
Does this work instead?

`.${classes.media}, .${classes.header}`

Reply:
It does, I'm changing it.